packages/components/modules/messages/GroupDetails/types.ts

```
@@ -0,0 +1,22 @@
+
+import { ChatRoomsQuery$data } from '../../../__generated__/ChatRoomsQuery.graphql'
+import { GroupDetailsQuery as GroupDetailsQueryType } from '../../../__generated__/GroupDetailsQuery.graphql'
+import { MembersListFragment$data } from '../../../__generated__/MembersListFragment.graphql'
+import { ProfileCardProps } from './ProfileCard/types'
+
+type GroupMembers = NonNullable<MembersListFragment$data['participants']>
+export type GroupMembersEdge = GroupMembers['edges'][number]
+
+export type GroupDetailsProps = {
+  allProfilesRef: ChatRoomsQuery$data
+  onBackButtonClicked: () => {}
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)